Project.toml

```
@@ -0,0 +1,4 @@
+name = "JuliaCtags"
+uuid = "1c25b4e4-f198-11e9-1949-65e68c540e8d"
+authors = []
+version = "0.1.0"
```

```suggestion
version = "0.3.0"
```

I know there have been at least 3 major changes to the tag so might as well.

Could also just do:
```suggestion
version = "1.0.0"
```